.spellcheck-en-custom.txt

```diff
@@ -120,3 +120,12 @@ XT
 XTX
 Xu
 YAML
+UX
+CWD
+EP
+codebase
+config
+docstring
+podman
+CLI
+CLI's
```

Comment:
This isn't something to block on, but if you have to update the PR anyway, it'd be good to run `make spellcheck-sort`.